pkg/resources/utils.go

```diff
@@ -313,6 +315,24 @@ func BuildDeploymentForPolicyController(instance *operatorv1alpha1.AuditLogging)
 				Spec: corev1.PodSpec{
 					ServiceAccountName:            OperandRBAC,
 					TerminationGracePeriodSeconds: &seconds30,
+					Affinity: &corev1.Affinity{
+						NodeAffinity: &corev1.NodeAffinity{
+							RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+								NodeSelectorTerms: []corev1.NodeSelectorTerm{
+									{
+										MatchExpressions: []corev1.NodeSelectorRequirement{
+											{
+												Key:      "beta.kubernetes.io/arch",
+												Operator: corev1.NodeSelectorOpIn,
+												Values:   []string{gorun.GOARCH},
```

Comment:
Does `gorun.GOARCH` return the full list of archs or just the one being used at runtime?

Reply:
Changed to var `architectureList = []string{"amd64", "ppc64le", "s390x"}`
Verified images spun up successfully:
audit-logging-fluentd-ds-djl2k                            1/1     Running                      0          5m1s
audit-logging-fluentd-ds-v7x87                            1/1     Running                      0          4m58s
audit-logging-fluentd-ds-w89bf                            1/1     Running                      0          4m58s
audit-policy-controller-d66476dd5-8wdfq                   1/1     Running                      0          5m44s